extensions/standard-processors/tests/unit/ManifestTests.cpp

```diff
@@ -150,3 +151,24 @@ TEST_CASE("Test Scheduling Defaults", "[schedDef]") {
     }
   }
 }
+
+TEST_CASE("Test operatingSystem Defaults", "[opsys]") {
+  minifi::state::response::DeviceInfoNode manifest("minifi-system");
+  auto serialized = manifest.serialize();
+  REQUIRE(serialized.size() > 0);
+  minifi::state::response::SerializedResponseNode proc_0;
+  for (const auto &node : serialized) {
+    if ("systemInfo" == node.name) {
+      for (const auto &sinfo : node.children) {
+        if ("operatingSystem" == sinfo.name) {
+          proc_0 = sinfo;
+        }
+      }
+
```

Comment:
break?

Reply:
Oh yeah there's no reason to not stop. There won't be more than one but yah a break (doh). thanks!